### PR TITLE
Move more run-time actions to setup-time

### PR DESCRIPTION
**Pull request by bmc-msft**
This script moves more of the run-time actions to setup-time.  This is important for running fuzzing within docker containers, such that installing llvm & gdb is done as part of the container, rather than on each launch.

### CODE REVIEW

File: src/runtime-tools/linux/run.sh [hunk 1]
```diff
@@ -7,53 +7,35 @@ set -ex
...
-        fi
-        export RUST_BACKTRACE=full
         onefuzz-supervisor run --config /onefuzz/config.json
     ;;
     "repro")
         logger "onefuzz: starting repro"
-        if type apt > /dev/null 2> /dev/null; then
-            sudo apt install -y gdb gdbserver
-        fi
+        echo repro
```
Comment (ranweiler) [inline src/runtime-tools/linux/run.sh:38]: I forget, are these `echo` calls load-bearing?

File: src/runtime-tools/linux/setup.sh [hunk 2]
```diff
@@ -84,6 +85,17 @@ fi
 
 chmod -R a+rx /onefuzz/tools/linux
 
+if type apt > /dev/null 2> /dev/null; then
+    sudo apt update
+    sudo apt install -y gdb gdbserver
```
Comment (ranweiler) [inline src/runtime-tools/linux/setup.sh:90]: Does this mean we're now installing `gdbserver` on all `apt` VMs?

**Review by ranweiler — APPROVED** [review]

File: src/runtime-tools/linux/setup.sh (hunk 2 above)
Comment (bmc-msft) [reply src/runtime-tools/linux/setup.sh:90]: Yes.  We'll need to do this anyways once we move to performing live-repro in managed scalesets.

File: src/runtime-tools/linux/run.sh (hunk 1 above)
Comment (bmc-msft) [reply src/runtime-tools/linux/run.sh:38]: Nope, purely informative.

**Comment by bmc-msft** [comment]
Integration tests passed

**Pull request merged**